### DISCUSSION TITLE
Secret condition of YOLO achievement?

### DISCUSSION THREAD

**Discussion by k0kubun** [community/community#18159] [Other Feature Feedback, Questions, & Ideas] [answered]
> https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

Many people have a YOLO achievement. It seems to be about "Merged without a review", which I've done so many times in my own repositories, but it's not unlocked for me. I guess there's some extra condition you need to satisfy to get the achievement?

**Comment by bailey** [comment, marked as answer]
Hi @k0kubun, not all of our achievements are backfilled to include all of your historic contributions. Some will be more like easter eggs that you'll stumble upon later. ✨

**Reply by k0kubun** [reply]
Thank you for your answer :) I'm looking forward to getting more someday.

**Reply by adriano66** [reply]
Same here, this thing is bugged as the linked PR in the achievement on my profile is clearly approved.

**Reply by SwiftyProgrammer690** [reply]
Yes

**Reply by jonypeixoto** [reply]
All right!

**Comment by tanhongit** [comment]
I have obtained it and here is the explanation I got: "**Merged without a review**"

**Reply by goswamimohit** [reply]
yaa same with me.

**Reply by lucianoaraus** [reply]
That's perfectly what YOLO means 😂

**Reply by kyuna0312** [reply]
yeaaahhhh kyaaaahh samee hereeeee :3

**Reply by kyuna0312** [reply]
> That's perfectly what YOLO means joy

yeaahhhhhh :3 UmU

**Reply by ghost** [reply]
yesss

**Reply by jonypeixoto** [reply]
Perfect!

**Comment by StoneCypher** [comment]
@bailey - is backfill planned in the future?  sure would be nice

**Comment by KainingYing** [comment]
I want it!

**Reply by goswamimohit** [reply]
I got it yesterday but do not know the reason behind them.

**Reply by yassinbenmansour** [reply]
how ?

**Comment by goswamimohit** [comment]
I am newbie. Here can anyone please explain me? What it means by '`Merged without a review`? #18245

**Reply by Jannchie** [reply]
When you make a PR, you need to have the project maintainer **review** it, and in general you need to wait for the reviewer to **approve** your PR before it is merged into the main branch, but sometimes they merge it without using the github **approve** feature.

**Comment by nine03** [comment]
Hello I'm a beginner at using github
I've been trying it all day, but it didn't work out 
Can you tell me how to get YOLO achievement?
Can I not get it because I set the repository to private?

**Reply by chakravala** [reply]
getting the achievement just for its own sake is kind of silly, and is going to pollute github with this kind of useless activity

**Reply by irezaul** [reply]
yes ! sounds good .....

**Reply by abdullahonus** [reply]
so did u got it already?

**Reply by Wjplatformer** [reply]
> getting the achievement just for its own sake is kind of silly, and is going to pollute github with this kind of useless activity

<br>

to be honest I am guilty of this (sort of heh) well thanks for telling me that, sure to take note of that @chakravala

**Reply by yassinbenmansour** [reply]
how to get the achievement

**Comment by yoonbitnara** [comment]
Merged without a review ? Really?

**Reply by SwiftyProgrammer690** [reply]
exactly

**Reply by abdullahonus** [reply]
is taht all

**Reply by sibeux** [reply]
what that mean?

**Comment by kotori-y** [comment]
an interestion achievement 😄

**Comment by wallaceleonel** [comment]
Achei interessante contudo quero saber como completa as outras tarefas para ganhar o restante rsrsrs  😄

**Reply by Wjplatformer** [reply]
Para a conquista Pull Shark, você precisa ter 2 pull requests mesclados. Para as conquistas do Galaxy Brain, você precisa fazer 2 respostas aceitas. Mais de [aqui](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by jayedrashid** [comment]
Merged without a review is too unprofessional.

**Comment by Satosh-J** [comment]
I also got this achievement, is it good? Merged without review is not good action I think.

**Comment by SpaceBuckett** [comment]
You get this achievement unlocked once your pull request is assigned to some reviewer and he/she merge your code without a review. 

I hope you unlock this achievement soon 🚀

**Comment by Satosh-J** [comment]
Hello, Everyone.
I think the Yolo achievement is gold.
Meaning merged without review means that the reviewer always trust my pull request and believe my code is good.
So the reviewer feels no need to test my code,
Any opinion?

**Reply by ramonpaolo** [reply]
maybe

**Comment by mainframev** [comment]
damn, I'm doing it every day, where is my achievement :D I want it ❤️

**Comment by pablodz** [comment]
Want it x2

**Comment by dyollb** [comment]
strange that people want YOLO, since it doesn't seem positive to me.

also, I got the badge for merging a PR without a review [here](https://github.com/dyollb/sitk-cli/pull/3)
- with changes in a readme file
- in my own (public) repo with no collaborators
- and my pull_request action [ignores changes in *.md](https://github.com/dyollb/sitk-cli/blob/3b4420446df5a9a00cc4353af206aea51735f594/.github/workflows/deploy_and_test.yml#L8) 
-> this feature just is annoying

**Comment by vicvenpet** [comment]
I found this other [discussion](https://github.com/Schweinepriester/github-profile-achievements/discussions/6) on github answered and tested by users, I hope you find it useful! 🤟

**Comment by FerdawsSaid** [comment]
How is it an achievement anyways! I perceive it kinda negative or lazy to merge code without reviewing xD
